### PR TITLE
kv: log warning if get timestamp too slow.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by morefreeze** [comment]
LGTM

File: kv.go [hunk 1]
```diff
@@ -152,10 +152,15 @@ func (s *tikvStore) UUID() string {
 }
 
 func (s *tikvStore) CurrentVersion() (kv.Version, error) {
+	now := time.Now()
 	startTS, err := s.oracle.GetTimestamp()
 	if err != nil {
 		return kv.NewVersion(0), errors.Trace(err)
 	}
+	dist := time.Since(now).Nanoseconds() / int64(time.Millisecond)
```
Comment (siddontang) [inline kv.go]: dist = time.Since(now)
if dist > 30 * time.Millisecond 

btw, 30ms seems a little short.

Comment (ngaut) [inline kv.go]: Every transaction need to get timestamp twice. 30 is kind of long enough.

**Comment by disksing** [comment]
This function is rarely used. I think you put this into pd-client.

**Comment by ngaut** [comment]
PTAL @disksing @morefreeze

File: kv.go [hunk 2]
```diff
@@ -156,6 +156,7 @@ func (s *tikvStore) CurrentVersion() (kv.Version, error) {
 	if err != nil {
 		return kv.NewVersion(0), errors.Trace(err)
 	}
+
```
Comment (disksing) [inline kv.go]: Please remove this empty line. The rest LGTM.

**Comment by ngaut** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**